CoseHandler/ValidationResult.cs

```diff
@@ -93,11 +101,13 @@ public void AddError(ValidationFailureCode code)
             certDetails = certDetailsBuilder.ToString();
         }
 
+        string contentValidationTypeMsg = "Payload validation type: " + ContentValidationType;
+
         if (Success)
         {
             // Print success. If verbose, include any chain validation messages.
             return ((verbose && InnerResults != null) ? $"Validation succeeded.{newline}{string.Join(newline, InnerResults.Select(r => r.ResultMessage))}" :
-                $"Validation succeeded.") + $"{newline}{certDetails}";
+                $"Validation succeeded.") + $"{newline}{certDetails}{newline}{contentValidationTypeMsg}{newline}";

```

Comment:
Do we want the extra content and newlines in regular detached and embedded validation?
If not, I suggest on line 102 checking to see if there is a contentValidationTypeMsg and appending it to certDetails when present